src/index.html

```diff
@@ -2,13 +2,47 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo-link" href="#">
+        <img src="./images/logo.png" alt="MOYO Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="/">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/" data-qa="hover">Laptops&nbsp;&&nbsp;Computers</a>
```

Comment:
Replace `&nbsp;` with a proper space or use CSS for spacing. Using `&nbsp;` can cause issues with responsiveness and is not the best choice for controlling layout or spacing. 🚀